unison-src/transcripts/squash.output.md

```diff
@@ -95,18 +95,16 @@ Let's try again, but using a `merge.squash` (or just `squash`) instead. The hist
 
   Nothing changed as a result of the merge.
 
-  😶
-  
-  builtin2 was already up-to-date with mybuiltin.
-
 .> history builtin2
 
   Note: The most recent namespace hash is immediately below this
         message.
   
+  ⊙ #1vqpv7b08f
```

Comment:
Same here, somewhat curious behaviour.

Reply:
I think we're now just always adding a new cons for the merge regardless of whether there are no changes; I wonder if a child is being updated strangely causing a new hash and therefor not "identical" branches?

Reply:
This is me keeping track of my musings so that I remember what I was thinking when I get back into things on Monday 😓  😛